### PR TITLE
Roblox.RobloxStudio version 1.6.0.29627

### PR DESCRIPTION
**Pull request by markandpreston1**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/39867)

### CODE REVIEW

**Comment by ItzLevvie** [comment]
About the `for WinGet` part: your users will see that even if they don't use WinGet to install it.

Here I double-clicked on "RobloxStudioLauncherBeta.exe":
image
image

Also, could it be possible for you to add a DisplayVersion for the installer to the registry? This should prevent upgrade loops for WinGet.

**Comment by markandpreston1** [comment]
> About the `for WinGet` part: your users will see that even if they don't use WinGet to install it.
> 
> Here I double-clicked on "RobloxStudioLauncherBeta.exe": image image
> 
> Also, could it be possible for you to add a DisplayVersion for the installer to the registry? This should prevent upgrade loops for WinGet.

how to do it?

**Comment by markandpreston1** [comment]
Just tell me the values to modify

**Comment by markandpreston1** [comment]
i tried importing the YAML file but it said a warning
image

**Comment by ItzLevvie** [comment]
Roblox Studio isn't open-sourced, so I won't be able to help you there, but I believe it's using a custom installer. You won't be able to make those changes in the package.

If you contact someone who maintains or has access to the bootstrapper source code, you should be able to tell them to use the [bootstrapper version](https://clientsettingscdn.roblox.com/v2/client-version/WindowsStudio64) for DisplayVersion.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

File: manifests/r/Roblox/RobloxStudio/1.6.0.29627/Roblox.RobloxStudio.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Roblox.RobloxStudio
+PackageVersion: 1.6.0.29627
```
Comment (ItzLevvie) [inline manifests/r/Roblox/RobloxStudio/1.6.0.29627/Roblox.RobloxStudio.installer.yaml:4]:
```suggestion
PackageVersion: 1.6.0.425477
```

File: manifests/r/Roblox/RobloxStudio/1.6.0.29627/Roblox.RobloxStudio.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Roblox.RobloxStudio
+PackageVersion: 1.6.0.29627
```
Comment (ItzLevvie) [inline manifests/r/Roblox/RobloxStudio/1.6.0.29627/Roblox.RobloxStudio.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 1.6.0.425477
```

File: manifests/r/Roblox/RobloxStudio/1.6.0.29627/Roblox.RobloxStudio.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: Roblox.RobloxStudio
+PackageVersion: 1.6.0.29627
```
Comment (ItzLevvie) [inline manifests/r/Roblox/RobloxStudio/1.6.0.29627/Roblox.RobloxStudio.yaml:4]:
```suggestion
PackageVersion: 1.6.0.425477
```

File: manifests/r/Roblox/RobloxStudio/1.6.0.29627/Roblox.RobloxStudio.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Roblox.RobloxStudio
+PackageVersion: 1.6.0.29627
+Installers:
+- Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://setup.rbxcdn.com/RobloxStudioLauncherBeta.exe
+  InstallerSha256: FF62991C65F22AC14AA9C53AB70D2D720A3C8A628E71CB305F46478310C513FD
```
Comment (ItzLevvie) [inline manifests/r/Roblox/RobloxStudio/1.6.0.29627/Roblox.RobloxStudio.installer.yaml:9]:
```suggestion
  InstallerSha256: AB515EF53C06C172329CB0DCAF75DB7291F62BCED759D6D24CE2DF1F186DBE0B
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Pull request closed without merging**